demos/asp-net-core/ITOps.ViewModelComposition/AssemblyLoader.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Extensions.DependencyModel;
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Loader;
+
+namespace ITOps.ViewModelComposition
+{
+    public static class AssemblyLoader
+    {
+        public static Assembly Load(string assemblyFullPath)
```

Comment:
yes and no, it's a generic way to load assemblies in .NET Core not connected to what we do here, it should go in its own "helper" assembly. It is required if we want (as I did for the DDD SouthWest demo) to introduce ViewComponents that are a very interesting Mvc Core feature for UI Composition